source/features/quick-mention.css

```diff
@@ -4,12 +4,11 @@
 }
 
 .rgh-quick-mention {
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	width: 100%;
-	height: 30px;
-	margin-bottom: -30px; /* Avoids pushing app badges down #2630 */
+	/* !important required to override .tooltipped */
+	position: absolute !important;
+	top: 100%;
+	left: 25%;
```

Comment:
25% is arbitrary, it should be 50% - half the width. This is generally achieved via:

```css
left: 50%;
translate: -50%;
```